src/generators/css-filter.ts

```diff
@@ -5,12 +5,22 @@ import {parseArray, formatArray} from '../utils/text';
 import {compareURLPatterns, isURLInList} from '../utils/url';
 import {createTextStyle} from './text-style';
 import {FilterConfig, InversionFix} from '../definitions';
+import { compareChromeVersions, getChromeVersion, isChrome } from '../utils/platform';
 
 export enum FilterMode {
     light = 0,
     dark = 1
 }
 
+export function hasRootIssue() {
```

Comment:
We both know about the issue, but let's name this function so that it makes sense for those who not. But not too long. Also you can add a link to the Chromium issue in comment.